contrib/externs/google_recaptcha_api.js

```diff
@@ -0,0 +1,37 @@
+
+/**
+ * Copyright 2016 ZapGroup LTD
```

Comment:
We do have things like "YouTube LLC" :)
https://github.com/google/closure-compiler/blob/master/contrib/externs/youtubeplayer.js

But yeah, "The Closure Compiler Authors" might be better.